include/itkImageToTorchImageFilter.h

```diff
@@ -89,19 +89,15 @@ class ITK_TEMPLATE_EXPORT ImageToTorchImageFilter : public ImageToImageFilter< T
   ImageToTorchImageFilter() = default;
   ~ImageToTorchImageFilter() override = default;
 
-  void PrintSelf( std::ostream &os, Indent indent ) const override;
+  void PrintSelf( std::ostream &os, Indent indent ) ITKv5_CONST override;
 
-  void GenerateInputRequestedRegion() override;
+  void VerifyPreconditions() ITKv5_CONST override;
 
-  void BeforeThreadedGenerateData() override;
+  void GenerateData() override;
 
   void DynamicThreadedGenerateData( const typename Self::OutputTorchImageRegionType &outputRegion ) override;
```

Comment:
This method is no longer needed.

Reply:
Right you are!

Reply:
Without DynamicThreadedGenerateData, the ctest fails with

> Description: itk::ERROR: itk::ERROR: ImageToTorchImageFilter(0x5647274e2070): Subclass should override this method!!! If old behavior is desired invoke this->DynamicMultiThreadingOff(); before Update() is called. The best place is in class constructor.

... so I have restored the override with the latest pushed commit.  Thoughts?

Reply:
Inside `GenerateData()` don't call [superclass' `GenerateData()`](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.1.1/Modules/Core/Common/include/itkImageSource.hxx#L210-L245). That's what is calling `DynamicThreadedGenerateData`. Instead, call `AllocateOutputs()` if you need it.

Reply:
Thank you.